pfsagentd/log.go

```diff
@@ -42,8 +44,8 @@ func logf(level string, format string, args ...interface{}) {
 		logMsg         string
 	)
 
-	enhancedFormat = "%-32s %-5s " + format
-	enhancedArgs = append([]interface{}{time.Now().Format(time.RFC3339Nano), level}, args...)
+	enhancedFormat = "[%s][%s][%d][%s] " + format
+	enhancedArgs = append([]interface{}{time.Now().Format(time.RFC3339Nano), globals.config.FUSEVolumeName, unix.Getpid(), level}, args...)
```

Comment:
I'd prefer you call os.Getpid() rather than unix.Getpid(): https://golang.org/pkg/os/#Getpid

Reply:
Done! Any thoughts on the change to the config file?